docs: Ringing doc fix (#494)
GetStream/stream-video-flutter: docusaurus/docs/Flutter/05-advanced/02-ringing.mdx

  }
}

@override
void dispose() {
  // ...
  _callKitEventSubscriptions.cancelAll();
}
```

Please add an additional extension at the end of the class / in a different file that we added to make
subscriptions simpler:

```dart
extension on Subscriptions {
  void addAll<T>(Iterable<StreamSubscription<T>?> subscriptions) {
    for (var i = 0; i < subscriptions.length; i++) {
      final subscription = subscriptions.elementAt(i);
      if (subscription == null) continue;

      add(i + 100, subscription);
    }
  }
}
```

The code upto this point handles calls for the background and foreground state of the app.
To handle calls from a terminated state, we need to add some additional code.

In a high-level widget, add this method and call it from the `initState()` method:

```dart
@override
void initState() {
  //...
  _tryConsumingIncomingCallFromTerminatedState();
}

void _tryConsumingIncomingCallFromTerminatedState() {
  if (_navigatorKey.currentContext == null) {
    // App is not running yet. Postpone consuming after app is in the foreground
    WidgetsBinding.instance.addPostFrameCallback((timeStamp) {
      _consumeIncomingCall();
    });
  } else {

>     // no-op. If the app is already running we'll handle this in didChangeAppLifecycleState

Reviewer: ```suggestion
    // no-op. If the app is already running we'll handle it via events
```
